src/components/AddItems.js

```diff
@@ -89,70 +92,85 @@ const AddItemsToList = () => {
   }
 
   return (
-    <div>
-      <div>
+    <div className="flex justify-center ">
+      <div className=" text-black md:mt-40 md:w-1/3 mt-20">
         <form onSubmit={submitShoppingListItemHandler}>
-          <h2>Add Item to List</h2>
           {shoppingListItemNameExists ? (
             <p>
               {`You have ${normalizeString(
                 shoppingListItemName,
               )} in your shopping list already`}
             </p>
           ) : null}
-          <div>
-            <label>
-              Name
+          <div className="">
+            <div className="flex justify-center">
```

Comment:
Why using flex here? When it is just about centering you can center it in the top with e.g. text-center